src/app/legacy/containers/Brand/index.jsx

```diff
@@ -30,8 +29,8 @@ const BrandContainer = ({ skipLink, scriptLink, brandRef, ...props }) => {
 
   return (
     <StyledBrand
-      backgroundColour={brandBackgroundColour}
-      logoColour={brandLogoColour}
+      backgroundColour={BRAND_BACKGROUND}
+      logoColour={BRAND_LOGO}
```

Comment:
Do you think its possible to use the theme colours within `StyledBrand` rather than passing them as props? Wouldn't worry if its too difficult to unpick, would just be cool to be able to isolate the styles right down to the elements using them.

Reply:
Ah yep good thinking!